README.md

```diff
@@ -21,6 +21,9 @@ This module has been tested to work on the following systems with Puppet v3.x on
  * Solaris 10
  * Ubuntu 10.04 LTS (Lucid Lynx)
  * Ubuntu 12.04 LTS (Precise Pangolin)
+ * OpenBSD 5.4
```

Comment:
Sounds good to me. Is just listing 'OpenBSD 5' ok? I don't think the minor versions really matter here, especially since this functionality has been around for ages.